homeassistant/components/smarthab/config_flow.py

```diff
@@ -0,0 +1,71 @@
+"""SmartHab configuration flow."""
+import logging
+
+import pysmarthab
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_EMAIL, CONF_PASSWORD
+
+from . import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class SmartHabConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """SmartHab config flow."""
+
+    def _show_setup_form(self, user_input=None, errors=None):
+        """Show the setup form to the user."""
+
+        if user_input is None:
+            user_input = {}
+
+        return self.async_show_form(
+            step_id="user",
+            data_schema=vol.Schema(
+                {
+                    vol.Required(
+                        CONF_EMAIL, default=user_input.get(CONF_EMAIL, "")
+                    ): str,
+                    vol.Required(CONF_PASSWORD): str,
+                }
+            ),
+            errors=errors or {},
+        )
+
```

Comment:
You'll need a `async_step_import`. If the schema is the same you can just do this:

```
    async def async_step_import(self, user_input):
        """Handle import."""
        return await self.async_step_user(user_input)
```